.circleci/config.yml

```diff
@@ -197,6 +197,38 @@ release-api-keys: &release-api-keys
       echo 'export GOOGLE_MAPS_API_KEY=$RELEASE_GOOGLE_MAPS_API_KEY' >> $BASH_ENV
       source $BASH_ENV
 
+set-beta-stage: &set-beta-stage
```

Comment:
We set a stage env_var for bugsnag to use, this way we'll be able to filter in the bugsnag dashboard by beta and release.